src/tests/test_piranha_swift.rs

```diff
@@ -32,22 +36,102 @@ create_rewrite_tests! {
     cleanup_comments = true,
     global_tag_prefix ="universal_tag.".to_string(),
     cleanup_comments_buffer = 3, delete_file_if_empty= false;
-  test_cleanup_rules_file: "cleanup_rules", 1,
-    substitutions = substitutions! {
-      "stale_flag" => "stale_flag_one",
-      "treated" => "true",
-      "treated_complement" => "false"
-    },
-    cleanup_comments = true, delete_file_if_empty= false;
   test_leading_comma: "leading_comma", 1,
     substitutions = substitutions! {
       "stale_flag" => "one"
     },
     cleanup_comments = true, delete_file_if_empty= false;
-  test_local_variable_inline_file: "variable_inline/local_variable_inline", 1,
-    cleanup_comments = true, delete_file_if_empty= false;
-  test_field_variable_inline_file: "variable_inline/field_variable_inline", 1,
-    cleanup_comments = true, delete_file_if_empty= false;
-  test_adhoc_variable_inline_file: "variable_inline/adhoc_variable_inline", 1,
-    cleanup_comments = true, delete_file_if_empty= false;
+}
+
+#[test]
+#[ignore] // Long running test
+fn test_cleanup_rules_file() {
+  super::initialize();
+  let _path = PathBuf::from("test-resources")
+    .join(SWIFT)
+    .join("cleanup_rules");
+  let temp_dir = super::copy_folder_to_temp_dir(&_path.join("input"));
+
+  let piranha_arguments = PiranhaArgumentsBuilder::default()
+    .path_to_codebase(temp_dir.path().to_str().unwrap().to_string())
+    .path_to_configurations(_path.join("configurations").to_str().unwrap().to_string())
+    .language(PiranhaLanguage::from(SWIFT))
+    .cleanup_comments(true)
+    .substitutions(substitutions! {
+      "stale_flag" => "stale_flag_one",
+      "treated" => "true",
+      "treated_complement" => "false"
+    })
+    .delete_file_if_empty(false)
+    .build();
+
+  execute_piranha_and_check_result(&piranha_arguments, &_path.join("expected"), 1, true);
+  // Delete temp_dir
+  temp_dir.close().unwrap();
+}
+
+#[test]
+#[ignore] // Long running test
+fn test_local_variable_inline_file() {
+  super::initialize();
+  let _path = PathBuf::from("test-resources")
+    .join(SWIFT)
+    .join("variable_inline/local_variable_inline");
+  let temp_dir = super::copy_folder_to_temp_dir(&_path.join("input"));
+
+  let piranha_arguments = PiranhaArgumentsBuilder::default()
+    .path_to_codebase(temp_dir.path().to_str().unwrap().to_string())
+    .path_to_configurations(_path.join("configurations").to_str().unwrap().to_string())
+    .language(PiranhaLanguage::from(SWIFT))
+    .cleanup_comments(true)
+    .delete_file_if_empty(false)
+    .build();
```

Comment:
This part repeats between the three tests below. Rather than a macro, my instinct here would be to have non-test helper function that you call as `make_default_piranha_arguments_with_path("variable_inline/local_variable_inline")` in this same file. Or even `execute_piranha_and_check_result_with_default_arguments(...)` if removing `temp_dir` is hard otherwise.

This is up to you, style-wise, though. Since for tests I'd usually lean on the side of test cases being clear vs avoiding repetition :)